.travis.yml

```diff
@@ -0,0 +1,31 @@
+language: php
+php:
+- '7.0'
```

Comment:
You can add as many as you like. Just takes longer because 2x the tests.

if you want we can add '7.0' and '7.1' but I know 'produciton' is on 7.0 right now.